adapters/bhero-market/index.js

```diff
@@ -0,0 +1,116 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BSC = require("../../sdk/binance");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const BigNumber = require("bignumber.js");
+
+class BHeroMarket {
+    constructor() {
+        this.name = "bhero-market";
+        this.symbol = "BCOIN";
+        this.token = "0x00e1656e45f18ec6747F5a8496Fd39B50b38396D";
+        this.protocol = "binance-smart-chain";
+        this.block = 18743562;
+        this.contract = "0x376a10e7f125a4e0a567cc08043c695cd8edd704";
+        this.events = ["Sold"];
+        this.pathToAbi = path.join(__dirname, "./abi.json");
+        this.range = 500;
+        this.chunkSize = 6;
+        this.sdk = undefined;
+    }
+
+    run = async () => {
+        this.sdk = this.loadSdk();
+        const token = await this.sdk.callContractMethod('bcoinContract');
```

Comment:
Notice that even though I've specified the token address on constructor I'm still fetching the token address from the contract, just to be sure

Reply:
Set it to `null`/`undefined` in the constructor then, please.

Reply:
Done [f59ea13](https://github.com/dappradar/nft-sales-adapters/pull/60/commits/f59ea13933251c3433063dd40f71c57b2c8685d2)

let me know if there is anything else